modules/core/misc/objc/common/IntVectorExt.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  IntVectorExt.swift
+//
+//  Created by Giles Payne on 2020/01/04.
+//
+
+import Foundation
+
+public extension IntVector {
+    convenience init(_ array:[Int32]) {
+        let data = array.withUnsafeBufferPointer { Data(buffer: $0) }
+        self.init(data:data);
+    }
+
+    subscript(index: Int) -> Int32 {
+        get {
+            self.get(index)
+        }
```

Comment:
> IntVectorExt.swift:18:9: error: missing return in a function expected to return

Observed on Mac Mini with OSX 10.14, Xcode 10.0.1.10010046 and CMake 3.15

Reply:
@alalek Implicit returns were introduced in Swift 5.1: https://github.com/apple/swift-evolution/blob/master/proposals/0255-omit-return.md

You'll need Xcode 11 or higher for Swift 5.1 support. All apps submitted to the App Store require Xcode 11 as of April 2020, so I don't think it's worth supporting earlier versions.

Reply:
@chrisballinger thanks for jumping in. I hadn't even realised this was a new feature. While the point about App Store development is a good one - a lot of people starting to dabble with OpenCV may be running older OS versions and Xcode versions so I think it makes sense to fix this